tests/test-recipes/metadata/_compile-test/f2_bad.py

```diff
@@ -1 +1 @@
-I am a bad file that should not pass compileall.
+I am a bad file that should not pass compileall
```

Comment:
Is there a reason for the changes in these small test-recipes sources? Did the missing `.` come from elsewhere?

Reply:
it was just an english correction as the line goes.. that is all